assets/base.css

```diff
@@ -1189,39 +1189,89 @@ deferred-media {
 .share-button {
   display: block;
   position: relative;
+  max-width: 31rem;
+}
+
+.share-button details {
+  width: fit-content;
+}
+
+.share-button__button {
+  font-size: 1.4rem;
+  display: flex;
+  align-items: baseline;
+  color: rgba(var(--color-link), var(--alpha-link));
+  margin-left: 0;
+  padding-left: 0;
+}
+
+.share-button__button:hover, details[open] > .share-button__button {
```

Comment:
Should we be adding a transition for an animation on hover since we have this for all other buttons? cc @Oliviammarcello

Reply:
Yes, good catch. Can we add it in @ludoboludo?

Reply:
Done 👌  [video](https://screenshot.click/21-08-1p70t-734fc.mp4)